### ISSUE TITLE
Documentation Link -> ERR_SSL_UNRECOGNIZED_NAME_ALERT

### ISSUE THREAD

**Issue by tmdevde** [spatie/laravel-permission#1972]
As the title says i get a ERR_SSL_UNRECOGNIZED_NAME_ALERT for the documentation link.

**Comment by erikn69** [comment]
https://spatie.be/docs/laravel-permission/v5/introduction looks ok

**Comment by tmdevde** [comment]
Yes but in this rep it's linked by [https://docs.spatie.be/laravel-permission](https://docs.spatie.be/laravel-permission). Just wanted to note that this link is not reachable.

**Comment by erikn69** [comment]
I didn't know that
